pkg/machines/libvirt.es6

```diff
@@ -56,6 +58,18 @@ if (!String.prototype.startsWith) {
     };
 }
 
+if (typeof String.prototype.includes === 'undefined') {
```

Comment:
Where is this required? Can't we find a way to do it via babel? Or avoid it?

Reply:
We could use https://babeljs.io/docs/usage/polyfill/

These are required here https://github.com/cockpit-project/cockpit/pull/8493/commits/3546515b488342698bda51c115bf3bfaf5c75264#diff-de184ca652af74b4142eac404126b77fR767 and in #7820 
- String: startsWith, includes
- Array: includes